.github/workflows/release.yml

```
@@ -16,14 +16,12 @@ jobs:
           submodules: recursive
 
       - name: 'Install Nix'
-        uses: cachix/install-nix-action@v15
+        uses: cachix/install-nix-action@v19
         with:
-          extra_nix_config: |
-            substituters = http://cache.nixos.org https://cache.iog.io
-            trusted-public-keys = cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ=
+          install_url: https://releases.nixos.org/nix/nix-2.13.3/install  
```

I think we should do this, because I've gotten warnings about getting rate-limited on github without being authenticated before:

```suggestion
          install_url: https://releases.nixos.org/nix/nix-2.13.3/install  
          extra_nix_config: |
            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

---

Can you make this change across all the PRs?